.gitignore

```diff
@@ -22,3 +22,6 @@ static/audio/**
 ecosystem.config.js
 .directory
 .idea/
+
+# Location for local files to be ignored
+/local
```

Comment:
What is "local files" and why is it prefixed with a slash? `/` implies it goes in the root folder, which wouldn't be committed anyways.